src/bpm/commands/run.go

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2018-Present CloudFoundry.org Foundation, Inc. All rights reserved.
+//
+// This program and the accompanying materials are made available under
+// the terms of the under the Apache License, Version 2.0 (the "License”);
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"bpm/config"
+	"bpm/exitstatus"
+	"bpm/models"
+	"bpm/runc/lifecycle"
+)
+
+func init() {
+	runCommand.Flags().StringVarP(&procName, "process", "p", "", "The optional process name.")
+	RootCmd.AddCommand(runCommand)
+}
+
+var runCommand = &cobra.Command{
+	Long:     "runs a bosh process synchronously",
```

Comment:
You can remove the `Long` if it is the same as the `Short`.

Reply:
ack